src/tools/auth0/handlers/clientGrants.ts

```diff
@@ -20,8 +20,14 @@ export const schema = {
   },
 };
 
+export type ClientGrant = {
```

Comment:
The `ClientGrant` and `ResourceServer` types were necessary to introduce because we're using properties of this type in the handler logic.